web/packages/teleport/src/config.ts

```diff
@@ -381,6 +381,10 @@ const cfg = {
     return cfg.auth.allowPasswordless;
   },
 
+  isAdminActionMFAEnforced() {
```

Comment:
```suggestion
  isAdminActionMfaEnforced() {
```

In the web code we only capitalize the first letter of an acronym.